src/webui/api/appcontroller.cpp

```diff
@@ -686,6 +692,11 @@ void AppController::setPreferencesAction()
     // Coalesce reads & writes
     if (hasKey("enable_coalesce_read_write"))
         session->setCoalesceReadWriteEnabled(it.value().toBool());
+#if (LIBTORRENT_VERSION_NUM >= 10202)
```

Comment:
Maybe this isn't really required? Users can set the option on/off but it won't have real effects.